package.json

```diff
@@ -24,7 +24,9 @@
     "test-debug": "cross-env NODE_ENV=test node --inspect-brk node_modules/jest/bin/jest.js --runInBand --verbose --config jest.config.js",
     "flow": "flow",
     "prettier": "prettier --write *.js *block-management/*.js",
-    "prettier-check": "prettier -l *.js *block-management/*.js"
+    "prettier-check": "prettier -l *.js *block-management/*.js",
+    "clean": "watchman watch-del-all; rm -rf node_modules; rm -f yarn.lock; rm -rf $TMPDIR/react-*; rm -rf $TMPDIR/metro-cache-*",
```

Comment:
How about cleaning the Jest cache as well? Command for that is `yarn test --clearCache` or `rm -Rf $TMPDIR/jest_*`.

Reply:
👍 I'll add this

Reply:
Updated in 55c951c  - I added it before removing `node_modules`, but it might fail if the jest module is not installed, but won't make the full clean task fail.

Reply:
Ah, I see. I don't think that anything goes wrong with the cleanup process if the node_modules dir is missing (for example, when running `yarn clean` twice one after the other) but cool, let's leave it as is 👍 .